examples/pxScene2d/src/pxScene.cpp

```diff
@@ -270,7 +276,15 @@ class sceneWindow : public pxWindow, public pxIViewContainer
   #endif
 
     context.term();
+    script.pump();
     script.collectGarbage();
+    #ifndef WIN32
+    sleep(5);
+    #endif
+    gUIThreadQueue.process(1.0);
+    #ifndef WIN32
+    sleep(5);
```

Comment:
Let's try to remove these sleeps since all the javascript is processed on this thread.